thingstodo.txt

```diff
@@ -0,0 +1,5 @@
+Add back rickmay statue
```

Comment:
Is this file supposed to be pushed up? If so, can this TODO be put in the main README instead?

Reply:
yes